src/IO/STL.cpp

```diff
@@ -1395,11 +1397,13 @@ int STLWriter::writeBegin(WriteMode writeMode)
 
     std::ios_base::openmode openMode;
     if (format == FormatBinary) {
-        if (writeMode != WriteOverwrite) {
+        if (writeMode == WriteOverwrite) {
+            openMode = std::ofstream::out | std::ofstream::binary;
+        } else if (partialWrite && writeMode == WriteAppend) {
```

Comment:
Binary STL files don't support multi-solid mode. WriteAppend mode is only valid in parallel cases where each processor writes a part of the facets.

Reply:
The method and the parameter are involved even in single-solid output in parallel case, right? 
However, if the parameter is needed for clarity and safeness for me it's ok.

Reply:
The function takes the partialWrite parameter regardless of whether bitpit is compiled with MPI support or not, because I think that having the ability to write binary files incrementally may be useful also for serial applications. For example to combine different bodies in a single binary STL.